cmd/src/debug_serv.go

```diff
@@ -0,0 +1,179 @@
+package main
```

Comment:
As above, this might be better named `debug_server.go`.

Reply:
fixed